### PR TITLE
Add "Introduction to Ansible" content to getting started

### PR DESCRIPTION
**Pull request by JaroslavKlech**
##### SUMMARY
Refreshes + moves the "About Ansible" content from https://docs.ansible.com/ansible/latest/index.html  and puts it as a high-level subsection right under the  "Getting started with Ansible" heading here https://docs.ansible.com/ansible/latest/getting_started/index.html

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com

##### ADDITIONAL INFORMATION
Closes issue #78459

### CODE REVIEW

**Comment by JaroslavKlech** [comment]
THis PR could fix also this issue https://github.com/ansible/ansible/issues/80941

**Comment by oraNod** [comment]
@JaroslavKlech Thanks for sending this PR and including the commit from GOVIND523. I suggest we move this to the new `ansible-documentation` repository so we can avoid the sanity tests for docs only PRs.

Your changes will still be published to `docs.ansible.com`. This repo change just allows us to build "on top of" the `ansible/ansible` repository. It also decouples the package docs from the core release cadence, which means we can have more community doc initiatives, give greater access and control of RST source to community, and iterate faster.

I'll move the corresponding issue but you'll need to do the following (please and thanks):

1. Fork https://github.com/ansible/ansible-documentation
2. Create a new branch against `ansible-documentation/devel`.
3. Add your `ansible/ansible` fork as a remote.
4. Cherry-pick the commits from this PR on to your branch.

You can find more details in this community topic: https://github.com/ansible-community/community-topics/issues/243

**Comment by oraNod** [comment]
@JaroslavKlech Another option might be to convert this PR to a draft, do all the work here, and then rebase on the new documentation repo. Things are not settled quite yet and I wouldn't want you to have to do yet another fork and jump through another set of hoops. :smile:

File: docs/docsite/rst/getting_started/index.rst [hunk 1]
```diff
@@ -1,99 +1,127 @@
...
 
-   .. code-block:: ansible-output
 
-      hosts (1):
-        192.0.2.50
-        192.0.2.51
-        192.0.2.52
+Setting up Ansible project
+--------------------------
+This guide will walk you through setting up a self-contained project structure for working with Ansible. The goal is to create a single directory that can easily be committed to a GitHub repository and used with utilities like AWX and Navigator. Additionally, we will integrate Visual Studio Code (VSCode) and Ansible Lint for an improved development experience.
```
Comment (samccann) [inline docs/docsite/rst/getting_started/index.rst:39]: @sivel made a series of comments on the other PR that included this content and I'm inclined to agree with those comments.

There's been a number of discussions of late on this.  @oraNod might recall better, but I think I   there is a 'recommended' directory structure that makes it easier to continue using playbooks with those other tools and that seems a worthwhile change here to the base getting started guide.

For other things -like integrating with vscode/lint etc - we should put that 'someplace else'.  Maybe it's the first page of  a new  'Getting Started with the Ansible Ecosystem' guide.

Comment (samccann) [reply docs/docsite/rst/getting_started/index.rst:39]: Also please review all the feedback from Please review the comments from https://github.com/ansible/ansible/pull/80978/ and apply here.

File: docs/docsite/rst/getting_started/index.rst [hunk 2]
```diff
@@ -1,99 +1,127 @@
-.. _getting_started_index:
+Getting Started with Ansible in a Self-Contained Project Structure
+==================================================================
```
Comment (samccann) [inline docs/docsite/rst/getting_started/index.rst:2]: If we move this to a new page (not overwrite the existing index page) then this title could be 'Creating a selfcontained project structure for Ansible' or something.

File: docs/docsite/rst/getting_started/index.rst [hunk 3]
```diff
@@ -1,99 +1,127 @@
...
 
-############################
-Getting started with Ansible
-############################
+Introduction to Ansible
+-----------------------
 
-Ansible automates the management of remote systems and controls their desired state.
-A basic Ansible environment has three main components:
+Ansible is an open-source automation platform that allows IT professionals to automate various areas of their domain, for example:
```
Comment (samccann) [inline docs/docsite/rst/getting_started/index.rst:8]: I'm not sure why we are repeating an intro to ansible here? Can't we just point to the existing intro page?

**Review by samccann — COMMENTED** [review]
I started making line by line comments but have stopped. As @sivel mentioned in the prior PR that had some of this content - this is fundamentally a different guide to the getting started guide. It removes the simplicity of getting a first playbook done. I'd rather see all of this as a separate new page. We can add it as part of the full getting started guide, but it shouldn't overwrite the getting_started/index.rst file as is done here.

File: docs/docsite/rst/getting_started/index.rst (hunk 1 above)
Comment (samccann) [reply docs/docsite/rst/getting_started/index.rst:39]: I see this PR is attempting to fix two related issues (and thus both the future-proof'd directory structure and vscode etc. Makes more sense now, but still think it's a separate page, not part of this first getting started page.

**Comment by JaroslavKlech** [comment]
@samccann, @oraNod I am closing this PR in favor of https://github.com/ansible/ansible-documentation/pull/14

File: docs/docsite/rst/getting_started/index.rst (hunk 3 above)
Comment (JaroslavKlech) [reply docs/docsite/rst/getting_started/index.rst:8]: Should be fixed in a new PR: https://github.com/ansible/ansible-documentation/pull/14

**Pull request closed without merging**